lib/engine/step/dividend.rb

```diff
@@ -82,7 +82,9 @@ def log_run_payout(entity, kind, revenue, action, payout)
         end
       end
 
-      def share_price_change(_entity, revenue)
+      def share_price_change(entity, revenue)
+        return {} if entity.minor?
```

Comment:
override this class rather than altering

Reply:
Isn't this applicable to all minors in all games?

Reply:
We have specific code for all other games that have minors, so it's best to stick with the same pattern.